lib/sprockets/uri_utils.rb

```diff
@@ -137,15 +138,24 @@ def parse_file_digest_uri(uri)
     # Examples
     #
     #   build("/tmp/js/application.js")
-    #   # => "file-digest:/tmp/js/application.js"
+    #   # => "file-digest:///tmp/js/application.js"
```

Comment:
Why not use `file-digest:/tmp/js/application.js` for absolute paths and `file-digest:tmp/js/application.js` for relative?